src/lib/progress-spinner/progress-spinner.html

```diff
@@ -5,5 +5,5 @@
 -->
 <svg viewBox="0 0 100 100"
      preserveAspectRatio="xMidYMid meet">
-  <path></path>
+  <path [style.strokeWidth]="strokeWidth"></path>
```

Comment:
If the value is being set in the style attribute here, should it be removed from the scss file?